### PR TITLE
docs: update ReadMe.md to include new content and CI repo link

### LINKED ISSUE #32: Update README

### LINKED ISSUE #61: Incomplete milestone v1.1 requirements
image

We need to remove all traces of AB3 and update the necessary files

Things to be done:

- [x] README
- [x] docs/AboutUs.md
- [x] docs/_config.yml
- [x] docs/UserGuide (no traces of AB3)
- [x] docs/DeveloperGuide  (no traces of AB3)

### PR DESCRIPTION
**Pull request by justgnohUG**
closes #32

### CODE REVIEW

**Comment by yaowei-soc** [comment]
LGTM, but perhaps might want to rename PR to something more appropriate. See #52

**Comment by justgnohUG** [comment]
@yaowei-soc fixed PR naming convention to provide more context and information.

Thanks for the reference.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+## Description
+
+This is **a project for bash-savvy users who are familiar with Command-Line-Interface.** We hope that with Bash-like commands, users who enjoy the high speed workflow of executing bash commands can be as seamless and easy to use in our project.
+
+## Usages
+* as a CLI integrated address book.
+* contact tracker to better manage your contacts and relationships.
+
+## Setup
```
Comment (yaowei-soc) [inline README.md:15]: Trailing white space

File: README.md [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+This is **a project for bash-savvy users who are familiar with Command-Line-Interface.** We hope that with Bash-like commands, users who enjoy the high speed workflow of executing bash commands can be as seamless and easy to use in our project.
+
+## Usages
+* as a CLI integrated address book.
+* contact tracker to better manage your contacts and relationships.
+
+## Setup 
+[coming soon]
+
+## Features
```
Comment (yaowei-soc) [inline README.md:18]: Trailing white space

**Review by yaowei-soc — CHANGES_REQUESTED** [review]
image

**Review by yaowei-soc — APPROVED** [review]
LGTM

**Pull request merged**